sdk/communication/azure-communication-administration/README.md

```diff
@@ -303,6 +303,22 @@ SyncPoller<Void, Void> res =
 res.waitForCompletion();
 ```
 
+### Release Phone Numbers
+<!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L399-L409 -->
+```java
+    PhoneNumber phoneNumber = new PhoneNumber("PHONE_NUMBER_TO_RELEASE");
+    List<PhoneNumber> phoneNumbers = new ArrayList<>();
+    phoneNumbers.add(phoneNumber);
+    PhoneNumberClient phoneNumberClient = createPhoneNumberClient();
+
+    SyncPoller<PhoneNumberRelease, PhoneNumberRelease> res = 
+        phoneNumberClient.beginReleasePhoneNumbers(phoneNumbers, duration);
+    res.waitForCompletion();
+    PhoneNumberRelease result = res.getFinalResult();
+    System.out.println("Phone number status: " + result.getStatus());
```

Comment:
Update this to "Phone number release status" ?

Reply:
Thank you!